README.md

```diff
@@ -50,6 +50,12 @@ server project page at http://www.freeipa.org/ .
 The most up-to-date documentation can be found at
 http://freeipa.org/page/Documentation .
 
+## For developers
+
+* Building FreeIPA from source
+    * Please, check http://www.freeipa.org/page/Build
+    * Please, check the BUILD.txt file at the root directory
```

Comment:
I'd drop the "Please, check" from each line, to look like:

* http://www.freeipa.org/page/Build
* See the BUILD.txt file in the source root directory